### PR TITLE
Pa/fix login error

### PR DESCRIPTION
**Pull request by palworth**
Fixing login error

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -56,10 +56,12 @@
   namespace :merchant do
   #only admin users will be able to reach this resource
   get '/dashboard', to: "dashboard#show"
+  get '/:merchant_id/dashboard', to: "dashboard#show"
```
Comment (palworth) [inline config/routes.rb:59]: adding route for story 34

File: db/schema.rb [hunk 2]
```diff
@@ -58,7 +58,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.bigint "user_id"
-    t.string "status"
+    t.string "status", default: "pending"
```
Comment (palworth) [inline db/schema.rb:61]: adding default

File: db/schema.rb [hunk 3]
```diff
@@ -80,7 +80,7 @@
     t.string "password_digest"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.integer "role", default: 0
+    t.integer "role", default: 1
```
Comment (palworth) [inline db/schema.rb:83]: changing default

File: spec/features/admins/access_to_merchant_dashbaord_show_spec.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# require 'rails_helper'
```
Comment (palworth) [inline spec/features/admins/access_to_merchant_dashbaord_show_spec.rb:1]: test for story.34

**Review by agallant121 — APPROVED** [review]

**Review by palworth — COMMENTED** [review]
Fixing login error

**Pull request merged**